src/codeu/chat/client/commandline/Chat.java

```diff
@@ -18,11 +18,14 @@
 import java.util.Map;
 import java.util.Scanner;
 import java.util.Stack;
+import java.io.*;
+import java.util.*;
```

Comment:
Don't do wildcard imports; they cause confusion in many cases. Let Eclipse take care of imports for you.